doc/classes/SceneTree.xml

```diff
@@ -42,7 +42,11 @@
 			<description>
 				Changes the running scene to the one at the given [param path], after loading it into a [PackedScene] and creating a new instance.
 				Returns [constant OK] on success, [constant ERR_CANT_OPEN] if the [param path] cannot be loaded into a [PackedScene], or [constant ERR_CANT_CREATE] if that scene cannot be instantiated.
-				[b]Note:[/b] The new scene node is added to the tree at the end of the frame. This ensures that both scenes aren't running at the same time, while still freeing the previous scene in a safe way similar to [method Node.queue_free]. As such, you won't be able to access the loaded scene immediately after the [method change_scene_to_file] call.
+				[b]Note:[/b]
+				Operations happen in the following order when [method change_scene_to_file] is called:
```

Comment:
```suggestion
				[b]Note:[/b] Operations happen in the following order when [method change_scene_to_file] is called:
```

Otherwise the word "Note" takes the whole paragraph.

Reply:
Done.